README.md

```diff
@@ -11,6 +11,11 @@ To download swiftly and install Swift, run the following in your terminal, then
 curl -L https://swiftlang.github.io/swiftly/swiftly-install.sh | bash
```

Comment:
We either need to edit this script to download the correct version of swiftly (Linux or mac) and run `swiftly init` or get rid of this line in the docs.

Reply:
This is tricky since this README is one of the places where people see how to install swiftly, but it's a few commits newer than the last release. Also, it's unclear if we are going to cut a release or not that still makes use of the shell script, which will still technically work.

Reply:
> reduced to copy the correct version of swiftly off the internet.

Yeah that's probably a good idea ... i think the maximum installation complexity i'd expect is "choose macOS or linux". Even that can probably be avoided in the script. Hopefully no manual architecture detection "are you on arm64 or amd64?!". manually choosing the arch is not the end of the world, but it can be harder than it looks, to beginners.